syncmaster/db/models/apscheduler_job.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+from sqlalchemy import Index, String
+from sqlalchemy.dialects.postgresql import BYTEA, DOUBLE_PRECISION
+from sqlalchemy.orm import Mapped, mapped_column
+
+from syncmaster.db.models.base import Base
+
+
+class APSchedulerJob(Base):
+    __tablename__ = "apscheduler_jobs"
+
+    id: Mapped[str] = mapped_column(String(191), primary_key=True, nullable=False)
```

Comment:
String as a primary key?

Reply:
Yes, this is apscheduler's requirement to store `id` as a string. 

![image](https://github.com/user-attachments/assets/1097d4cd-2b04-4273-81d2-7d40a9d742c4)


If you are wondering about efficiency, we could make BigInteger field as a primary key, but get_job() methods will still rely on that textual id

Reply:
We can change this later